docs/assets/yamlexamples/dls-example.yaml

```diff
@@ -1,73 +1,110 @@
+kind: MemoryBroker
 metadata:
   name: demo
-spec:
-  delivery:
-    deadLetterSink:
-      ref:
-          apiVersion: serving.knative.dev/v1
-          kind: Service
-          name: event-failure-capture
-    backoffDelay: "PT0.5S"     # or ISO8601 duration
-    backoffPolicy: exponential # or linear
-    retry: 2
 
 ---
 
-apiVersion: sources.knative.dev/v1
-kind: PingSource
+apiVersion: sources.triggermesh.io/v1alpha1
+kind: WebhookSource
 metadata:
-  name: say-hi
+  name: webhook
 spec:
-  data: '{"hello": "triggermesh"}'
-  schedule: "*/1 * * * *"
+  eventType: webhook.event
   sink:
     ref:
-      apiVersion: eventing.knative.dev/v1
-      kind: Broker
+      apiVersion: eventing.triggermesh.io/v1alpha1
+      kind: MemoryBroker
       name: demo
 
 ---
 
-apiVersion: serving.knative.dev/v1
-kind: Service
+apiVersion: apps/v1
+kind: Deployment
 metadata:
   name: event-success-capture
 spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: event-success-capture
   template:
     metadata:
-      annotations:
-        autoscaling.knative.dev/min-scale: "1"
+      labels: *labels
     spec:
       containers:
-      - image: gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
+        - name: event-display
+          image: gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
+
+---
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: event-success-capture
+spec:
+  selector:
+    app: event-success-capture
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 8080
 
 ---
 
-apiVersion: eventing.knative.dev/v1
+apiVersion: eventing.triggermesh.io/v1alpha1
 kind: Trigger
 metadata:
   name: demo-to-display
 spec:
-  broker: demo
-  subscriber:
+  broker:
+    group: eventing.triggermesh.io
+    kind: MemoryBroker
+    name: demo
+  target:
     ref:
-      apiVersion: serving.knative.dev/v1
+      apiVersion: v1
       kind: Service
       name: event-success-capture
+  delivery:
+    deadLetterSink:
+      ref:
+          apiVersion: v1
+          kind: Service
+          name: event-failure-capture
+    backoffDelay: "PT0.5S"     # ISO8601 duration
+    backoffPolicy: exponential # exponential or linear
+    retry: 2
 
 ---
 
-apiVersion: serving.knative.dev/v1
-kind: Service
+apiVersion: apps/v1
+kind: Deployment
 metadata:
   name: event-failure-capture
 spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: event-failure-capture
   template:
     metadata:
-      annotations:
-        autoscaling.knative.dev/min-scale: "1"
+      labels: *labels
     spec:
       containers:
-      - image: gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
+        - name: event-display
+          image: gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
+
+---
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: event-failure-capture
+spec:
+  selector:
+    app: event-failure-capture
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 8080
```

Comment:
```suggestion
    targetPort: 8080
    
```

Reply:
Is this (and other similar ones) fixing a formatting issue?